microovn/api/services/control.go

```diff
@@ -72,6 +72,18 @@ func disableService(s state.State, r *http.Request) response.Response {
 		logger.Errorf("Failed to get service: %s", err)
 		return response.ErrorResponse(500, "internal server error")
 	}
+
```

Comment:
Is this change supposed to be committed? It doesn't seem to have any effect aside from the log messasges.